manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
Exclude the `.idea` directory with underlying content from the PR, format `.gitignore` to ignore this directory and content as well.